ocrd_network/ocrd_network/rabbitmq_utils/connector.py

```diff
@@ -0,0 +1,262 @@
+        self._logger = logger
+        self._host = host
+        self._port = port
+        self._vhost = vhost
+
+        # According to the documentation, Pika blocking
+        # connections are not thread-safe!
+        self._connection = None
+        self._channel = None
+
+        # Should try reconnecting again
+        self._try_reconnecting = False
+        # If the module has been stopped with a
+        # keyboard interruption, i.e., CTRL + C
+        self._gracefully_stopped = False
+
+    @staticmethod
+    def declare_and_bind_defaults(connection: BlockingConnection, channel: BlockingChannel) -> None:
+        if connection and connection.is_open:
+            if channel and channel.is_open:
+                # Declare the default exchange agent
+                RMQConnector.exchange_declare(
+                    channel=channel,
+                    exchange_name=DEFAULT_EXCHANGER_NAME,
+                    exchange_type=DEFAULT_EXCHANGER_TYPE,
+                )
+                # Declare the default queue
+                RMQConnector.queue_declare(
+                    channel,
+                    queue_name=DEFAULT_QUEUE
+                )
+                # Bind the default queue to the default exchange
+                RMQConnector.queue_bind(
+                    channel,
+                    queue_name=DEFAULT_QUEUE,
+                    exchange_name=DEFAULT_EXCHANGER_NAME,
+                    routing_key=DEFAULT_ROUTER
+                )
+
+    # Connection related methods
+    @staticmethod
+    def open_blocking_connection(
+            credentials: PlainCredentials,
+            host: str = HOST,
+            port: int = PORT,
+            vhost: str = VHOST
+    ) -> BlockingConnection:
+        blocking_connection = BlockingConnection(
+            parameters=ConnectionParameters(
+                host=host,
+                port=port,
+                virtual_host=vhost,
+                credentials=credentials,
+                # TODO: The heartbeat should not be disabled (0)!
+                heartbeat=0
+            ),
+        )
+        return blocking_connection
+
+    @staticmethod
+    def open_blocking_channel(connection: BlockingConnection) -> Union[BlockingChannel, None]:
+        if connection and connection.is_open:
+            channel = connection.channel()
+            return channel
+        return None
+
+    @staticmethod
+    def exchange_bind(
+            channel: BlockingChannel,
+            destination_exchange: str,
+            source_exchange: str,
+            routing_key: str,
+            parameters: Optional[Any] = None
+    ) -> None:
+        if parameters is None:
+            parameters = {}
+        if channel and channel.is_open:
+            channel.exchange_bind(
+                destination=destination_exchange,
+                source=source_exchange,
+                routing_key=routing_key,
+                parameters=parameters
```

Comment:
Indeed it is `arguments` and that's the only place where it's `parameters`. It is safe to rename `parameters` to `arguments` here because the `exchange_bind` method is not invoked anywhere else yet.